### PR TITLE
fixes for people

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] better typing
- [x] less render thrash

I noticed this while working on other stuff on the people tab

@keybase/react-hackers

### CODE REVIEW

File: shared/people/container.js [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
+}
+
+class LoadOnMount extends React.PureComponent<Props> {
   componentDidMount() {
     this.props.getData(false)
   }
 
+  _onSearch = () => this.props.onSearch()
+  _getData = (markViewed?: boolean) => this.props.getData(markViewed)
+  _onClickUser = (username: string) => this.props.onClickUser(username)
```
Comment (buoyad) [inline shared/people/container.js]: why not pass these functions through directly?

Comment (buoyad) [reply shared/people/container.js]: oh, so `People` won't need to rerender if the functions change 👍

File: shared/people/container.js [hunk 2]
```diff
@@ -39,9 +66,9 @@ const mapDispatchToProps = (dispatch, {navigateAppend}) => ({
 
 const mergeProps = (stateProps, dispatchProps) => {
   return {
-    newItems: stateProps._newItems.toJS(),
-    oldItems: stateProps._oldItems.toJS(),
-    followSuggestions: stateProps.followSuggestions.toJS(),
+    newItems: stateProps._newItems,
+    oldItems: stateProps._oldItems,
```
Comment (buoyad) [inline shared/people/container.js]: we should get rid of these underscored names

Comment (chrisnojima) [reply shared/people/container.js]: done

**Pull request merged**